### DISCUSSION TITLE
ask for refund

### DISCUSSION THREAD

**Discussion by BobBeesheq** [aclap-dev/video-downloadhelper#916] [License & Purchase requests]
### What's your issue?

Hello. I use the Video DownloadHelper add-on mainly in the Opera GX browser, where I use the add-on designed for Chrome. When I tried to download the video it said to purchase a license. I have purchased a license. Thinking everything was fine, I wanted to download the video and it still asked me to purchase a license. I suddenly found out that I bought a license for Firefox. Unfortunately, I don't use Firefox, so I would like to request a refund. 
Thank you. 

My email: nutil.petr.ji@gmail.com

### The 10 first digits (only 10) of your license number or email address

1c3a89a3d7

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by VITI1305** [comment]
Hello, I want to be refunded the money I paid because when it is free I have been able to download what I needed, when I pay for the license it gives me errors and I get a message that this system does not support downloading. I also want to unsubscribe from the supposed renewals that there may be of that service. Thank you.

**Reply by jcfrog** [reply]
Hi,
to be refunded please send us the email address you used for purchase at licenses@downloadhelper.net
For the renewal, check your browser's extensions auto update options.

**Comment by boe308** [comment]
hi i purchased the wrong one i wanted the edge one i didnt know there is different ones for each browser
